prosemirror-jdita/src/plugin.ts

```diff
@@ -58,12 +125,35 @@ function markItem(mark: MarkType, props: Partial<MenuItemSpec> = {}): MenuElemen
   });
 }
 
+/**
```

Comment:
`Define interface ` SimpleItemCallbacks` to provide menu` -> `Menu`

Reply:
Fixed with 588cff1 - I'm not sure, if the current fix meets your suggestion?

Reply:
@adamretter A new suggestion in ae39f59, replacing the old phrase with `Menu`  made the comment bit ambiguous, it  came out to be: 
```js
/**
 * Menu item callback return types
 * with fixed and optional properties
  */
``` 
This is the updated comment header I suggested: 
```js
/**
 * Menu item callbacks
 */
interface SimpleItemCallbacks {
  call: () => void;
  enable?: () => boolean;
  active?: () => boolean;
}
```